src/pages/components/tags.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Tags
+description: >
+  A Tag is an interactive label used to categorize content.
+package: '@zendeskgarden/react-tags'
+components:
+  - tags/src/elements/Tag.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To categorize by dimension, attribute, or property</li>
+      <li>To represent filter parameters</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>To prompt an action, use a [Button](/components/button) instead</Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
```

Comment:
I'm curious, do we have some guiding principal for "How to use it" vs. "How to use"? In this case, there's a consideration that the singular "it" incorrectly refers to the plural "Tags". Consider https://garden.zendesk.com/components/icon-button where the heading is "How to use" (no "it") – although maybe that one was an oversight.
/cc @vsrixyz

Reply:
Good spot. From my perspective that's an oversight. I was operating on always going with `How to use it` and then subsequently `How to use it well`.

Reply:
Cool. I'll throw the page fix into the branch I'm currently working on.